src/aldente_client.cpp

```diff
@@ -101,13 +102,18 @@ void AldenteClient::start() {
 
     std::cerr << "Starting client..." << std::endl;
 
+    // Used for callbacks
+    Timer timer(GAME_TICK);
```

Comment:
Just confirming, so now the client's clock is the same duration as the server's?